### PR TITLE
[APIM 4.2.0] Global Key Manager Documentation

### PR DESCRIPTION
**Pull request by RusJaI**

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -1414,7 +1414,9 @@ nav:
             - Configure PingFederate as A Key Manager: administer/key-managers/configure-pingfederate-connector.md
             - Configure ForgeRock as a Key Manager: administer/key-managers/configure-forgerock-connector.md
             - Configure a Custom Key Manager: administer/key-managers/configure-custom-connector.md
+            - Configure the Global Key Manager: administer/key-managers/configure-global-key-manager.md
         - Advanced Configurations: administer/advanced-configurations.md
+        - Manage Role based access control for the Admin portal: administer/role-based-access-control.md
```
Comment (hisanhunais) [inline en/mkdocs.yml:1419]: Shall we check how this got added? This doesn't seem to be a part of the Global Key Manager docs

Comment (RusJaI) [reply en/mkdocs.yml]: It has been added from the cherry pick. Removed that line

File: en/docs/administer/key-managers/configure-global-key-manager.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+14. Click **GENERATE ACCESS TOKEN** to generate an application access token. Make sure to copy the generated JWT access token that appears so that you can use it in the future.
+
+15. Go to the Developer Portal landing page, select the SampleAPI and [Subscribe]({{base_path}}/consume/manage-subscription/subscribe-to-an-api/#subscribe-to-an-existing-application) to the SampleApp.
+
+    [![Subscribe SampleAPI to SampleApp]({{base_path}}/assets/img/administer/global-keymanager/subscribe-sample-api-to-sample-app.png)]({{base_path}}/assets/img/administer/global-keymanager/subscribe-sample-api-to-sample-app.png)
+
+16. [Try Out the API]({{base_path}}/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-the-integrated-api-console/) with the access token generated using the Global Key Manager.
+
+    A Successful response indicates that an API of the super tenant can be invoked using an access token generated for an application using the Global Key Manager.
```
Comment (SavinduDimal) [inline en/docs/administer/key-managers/configure-global-key-manager.md:107]: Shall we change 'S' in "Successful" to lower case?

**Review by hisanhunais — APPROVED** [review]

File: en/docs/administer/key-managers/configure-global-key-manager.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
+# Configure the Global Key Manager
+
+This document outlines the steps to enable the global key manager support in WSO2 API Manager (API-M). With this feature, a single token generated using the global key manager can be used to invoke cross-tenant APIs without generating a token in that specific tenant's key manager.
```
Comment (SavinduDimal) [inline en/docs/administer/key-managers/configure-global-key-manager.md:3]: Let's ensure consistency in the capitalization of 'global key manager' throughout the document.  Please use either 'global key manager' or 'Global Key Manager' consistently.

File: en/docs/administer/key-managers/configure-global-key-manager.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+9. Sign in to the WSO2 API Publisher (`https://<hostname>:9443/publisher`) using the new tenant’s (abc.com) admin credentials and repeat Steps 5 - 7.
+
+10. Sign in to the super tenant's Developer Portal using the super tenant’s admin credentials (`https://<hostname>:9443/devportal`).
+
+11. [Create an application]({{base_path}}/consume/manage-application/create-application/) (Ex: SampleApp).
+
+    [![Sample App Creation]({{base_path}}/assets/img/administer/global-keymanager/sample-app-creation.png)]({{base_path}}/assets/img/administer/global-keymanager/sample-app-creation.png)
+
+12. Click **Production Keys** and Select the Global Key Manager (IS-GKM) which was created from the list of key managers displayed.
```
Comment (SavinduDimal) [inline en/docs/administer/key-managers/configure-global-key-manager.md:93]: Shall we change 'S' in "Select the Global ..." to lower case?

File: en/docs/administer/key-managers/configure-global-key-manager.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+    [![Select Global Key Manager]({{base_path}}/assets/img/administer/global-keymanager/select-global-key-manager.png)]({{base_path}}/assets/img/administer/global-keymanager/select-global-key-manager.png)
+
+13. Click **GENERATE KEYS**.
+
+14. Click **GENERATE ACCESS TOKEN** to generate an application access token. Make sure to copy the generated JWT access token that appears so that you can use it in the future.
+
+15. Go to the Developer Portal landing page, select the SampleAPI and [Subscribe]({{base_path}}/consume/manage-subscription/subscribe-to-an-api/#subscribe-to-an-existing-application) to the SampleApp.
```
Comment (SavinduDimal) [inline en/docs/administer/key-managers/configure-global-key-manager.md:101]: Shall we change 
`Subscribe to the SampleApp.` to `Subscribe using the SampleApp.`

File: en/docs/administer/key-managers/configure-global-key-manager.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+    [![Tenant List]({{base_path}}/assets/img/administer/global-keymanager/tenant-list.png)]({{base_path}}/assets/img/administer/global-keymanager/tenant-list.png)
+
+19. Select the SampleAPI and Subscribe to the SampleApp similar to Step 15.
+
+    !!! Note
+        The keys generated for an application using the Global Key Manager will be displayed across tenants. In this example scenario, the keys for the sampleApp were generated in the Super Tenant’s Developer Portal. These keys are reflected in the Production Keys of the IS-GKM tab of the sampleApp in the abc.com’s Developer Portal.
+
+20. Try Out the API with the same access token generated using the Global Key Manager which was used in Step 16.
+
+A Successful response indicates that the API of a tenant can be invoked using an access token generated for an application using the Global Key Manager. Note that the access token was generated in the super tenant’s Developer Portal, but an API in a different tenant was invoked using this access token.
```
Comment (SavinduDimal) [inline en/docs/administer/key-managers/configure-global-key-manager.md:124]: Shall we change 'S' in "Successful" to lower case?

File: en/docs/administer/key-managers/configure-global-key-manager.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+
+This document outlines the steps to enable the global key manager support in WSO2 API Manager (API-M). With this feature, a single token generated using the global key manager can be used to invoke cross-tenant APIs without generating a token in that specific tenant's key manager.
+
+!!! Note
+      This feature is available only as an update and is available from Update level 4.2.0.82 (released on 13th May 2024).
+
+The global key manager can be created through the admin portal of the super tenant. This global key manager will be visible in the devportal of all the tenants and this can be used to generate a token which can be used to invoke APIs across tenants. Any third party key manager can be configured as a global key manager.
+
+!!! Note
+      It is only possible to create one global key manager which can only be done through the Admin portal of the super tenant.
```
Comment (SavinduDimal) [inline en/docs/administer/key-managers/configure-global-key-manager.md:11]: Shall we change `Admin portal` to `Admin Portal` to make it consistent through out the doc?

File: en/docs/administer/key-managers/configure-global-key-manager.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
+# Configure the Global Key Manager
+
+This document outlines the steps to enable the global key manager support in WSO2 API Manager (API-M). With this feature, a single token generated using the global key manager can be used to invoke cross-tenant APIs without generating a token in that specific tenant's key manager.
+
+!!! Note
+      This feature is available only as an update and is available from Update level 4.2.0.82 (released on 13th May 2024).
```
[the remark this answers is not in the document]
Comment (RusJaI) [reply en/docs/administer/key-managers/configure-global-key-manager.md:6]: fixed

File: en/docs/administer/key-managers/configure-global-key-manager.md (hunk 2 above)
Comment (RusJaI) [reply en/docs/administer/key-managers/configure-global-key-manager.md:107]: fixed

File: en/docs/administer/key-managers/configure-global-key-manager.md (hunk 3 above)
Comment (RusJaI) [reply en/docs/administer/key-managers/configure-global-key-manager.md:3]: fixed

File: en/docs/administer/key-managers/configure-global-key-manager.md (hunk 4 above)
Comment (RusJaI) [reply en/docs/administer/key-managers/configure-global-key-manager.md:93]: fixed

File: en/docs/administer/key-managers/configure-global-key-manager.md (hunk 6 above)
Comment (RusJaI) [reply en/docs/administer/key-managers/configure-global-key-manager.md]: fixed

File: en/docs/administer/key-managers/configure-global-key-manager.md (hunk 7 above)
Comment (RusJaI) [reply en/docs/administer/key-managers/configure-global-key-manager.md]: fixed

**Pull request merged**